contracts/utils/CHANGELOG.json

```diff
@@ -34,17 +34,12 @@
                 "pr": 1657
             },
             {
-<<<<<<< HEAD
                 "note": "Add unit tests for `LibAddressArray`",
                 "pr": 1712
             },
             {
                 "note": "Fix `LibAddressArray.indexOf` returning incorrect index.",
                 "pr": 1712
-=======
-                "note": "Change ReentrancyGuard implementation to cheaper one",
-                "pr": 1699
->>>>>>> Update PRs in changelogs.
```

Comment:
Looks like there are still some merge conflicts to resolve here, otherwise looks good to me.

Reply:
I think these are artifacts actually removed? Unless you're referring to something else in the file.

Reply:
Ah whoops, misread this.